x-pack/filebeat/magefile.go

```diff
@@ -100,8 +105,8 @@ func fieldsYML() error {
 	return devtools.GenerateFieldsYAML(devtools.OSSBeatDir("module"), "module", "input")
 }
 
-// Dashboards collects all the dashboards and generates index patterns.
-func Dashboards() error {
```

Comment:
Why removing this mage target? Should we do the change also in other beats?

Reply:
I do think we should make a similar change to all Beats to reuse the importable targets from dev-tools/mage/targets, but I'm not going to do this now. In fact I didn't mean to commit most of these changes. I was hacking at this to be able to import/export to Elastic Cloud.

I'm going to undo these and also fix the titles as you suggested.

Reply:
I left the changes to add KIBANA_URL support the export dashboard target, but I removed the refactoring of x-pack/filebeat/magefile.go.